src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/services/CoreCddCaseDataServiceTest.java

```diff
@@ -27,13 +27,6 @@
 @SpringBootTest(classes = TestCaseLoaderApp.class)
 public class CoreCddCaseDataServiceTest {
 
-    private static final String USER_TOKEN = "Bearer userToken";
-    private static final String S2S_TOKEN = "s2sToken";
-    private static final String USER_ID = "19";
-    private static final String JURISDICTION_ID = "SSCS";
-    private static final String CASE_TYPE_ID = "Benefit";
-    private static final String EVENT_ID = "appealCreated";
```

Comment:
Just out of interest, why has the test been changed to not test for these values?

Reply:
these values are loaded from the application.yaml now.